client/my-sites/upgrades/components/domain-warnings/pending-gapps-tos-notice.jsx

```diff
@@ -67,48 +71,91 @@ const PendingGappsTosNotice = React.createClass( {
 
 		switch ( severity ) {
 			case 'warning':
-				return this.translate( 'Attention!', translationOptions );
+				return this.props.translate( 'Attention!', translationOptions );
 
 			case 'error':
-				return this.translate( 'Urgent!', translationOptions );
+				return this.props.translate( 'Urgent!', translationOptions );
 
 			default:
-				return this.translate( 'You\'re almost there!', translationOptions );
+				return this.props.translate( 'You\'re almost there!', translationOptions );
 		}
 	},
 
 	generateLogInClickHandler( { domainName, user, severity, isMultipleDomains } ) {
 		return () => {
-			this.recordEvent( 'pendingAccountLogInClick', { siteSlug: this.props.siteSlug, domainName, user, severity, isMultipleDomains, section: this.props.section } );
+			this.recordEvent( 'pendingAccountLogInClick', {
+				domainName,
+				isMultipleDomains,
+				severity,
+				user,
+				section: this.props.section,
+				siteSlug: this.props.siteSlug
+			} );
+		};
+	},
+
+	generateFixClickHandler() {
+		return () => {
+			this.recordEvent( 'fixPendingEmailSiteNoticeClick', this.props.siteSlug );
 		};
 	},
 
+	compactNotice() {
+		const severity = this.getNoticeSeverity(),
+			href = this.props.domains.length === 1
+				? paths.domainManagementEmail( this.props.siteSlug, this.props.domains[ 0 ].name )
+				: paths.domainManagementEmail( this.props.siteSlug );
+
+		return (
+			<Notice
+				isCompact={ this.props.isCompact }
+				status={ `is-${ severity }` }
+				showDismiss={ false }
+				key="pending-gapps-tos-acceptance-domain-compact"
+				text={ this.props.translate(
+					'Email requires action',
+					'Emails require action',
+					{
+						count: this.props.domains.length
+					}
+				) }>
+				<NoticeAction
+					href={ href }
+					onClick={ this.generateFixClickHandler() }>
```

Comment:
This is a good example where we would see some benefit, bringing `>` to a new line would allow `{ this.props.translate( 'Fix' ) }` to be tabbed back in one tab like so:

```jsx
<NoticeAction
    href={ href }
    onClick={ this.generateFixClickHandler() }
>
        { this.props.translate( 'Fix' ) }
</NoticeAction>

```